kirby/models/__init__.py

```diff
@@ -150,6 +150,26 @@ class Schedule(db.Model):
     def add_suspension(self, suspension):
         self.suspensions.append(suspension)
 
+    @validates("hour", "minute", "day", "month", "weekday")
```

Comment:
According to my experience, `validates` only operates during INSERT and not during UPDATE, so I could input something correct, but break it afterwards. Have you tested updating?

Reply:
I added a test on UPDATE :

```python
    [...]
    schedule = (
        db.session.query(Schedule).filter_by(name="test schedule").one()
    )
    with raises(ValueError):
        schedule.minute = "/3"
        db.session.commit()
```

The test pass and raise an error when the `raises` is absent, so it is working.